hi.js

```diff
@@ -922,61 +922,61 @@ chall.name.cc Get {0} close-calls {1}
 modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
-commands.login Login to use this command
+commands.login इस कमांड को इस्तिमल करने के लिए लॉगिन करें
 commands.connected {0} connected, {1} in menu
-commands.host {0} is the current Host.
-commands.noclip.on Noclip - Enabled
-commands.noclip.off Noclip - Disabled
-commands.god.on God mode - Enabled
-commands.god.off God mode - Disabled
-commands.god.on.other God mode - Enabled for {0}
-commands.god.off.other God mode - Disabled for {0}
-commands.smite.on Smite - Enabled
-commands.smite.off Smite - Disabled
-commands.smite.on.other Smite - Enabled for {0}
-commands.smite.off.other Smite - Disabled for {0}
-commands.perk Gave {0} {1}
+commands.host {0} मौजूदा होस्ट हैं.
+commands.noclip.on Noclip - इनेबल्ड
+commands.noclip.off Noclip - डिसेबल्ड
+commands.god.on गॉड मोड - इनेबल्ड
+commands.god.off गॉड मोड - डिसेबल्ड
+commands.god.on.other गॉड मोड - {0} के लिए इनेबल्ड
+commands.god.off.other गॉड मोड - {0} के लिए डिसेबल्ड
+commands.smite.on Smite - इनेबल्ड
+commands.smite.off Smite - डिसेबल्ड
+commands.smite.on.other Smite - {0} के लिए इनेबल्ड
+commands.smite.off.other Smite - {0} के लिए डिसेबल्ड
+commands.perk दिया {0} {1}
 commands.maxperk Gave {0} MAX {1}
-commands.unlimited.on Unlimited Ammo - Enabled
-commands.unlimited.off Unlimited Ammo - Disabled
-commands.unlimited.on.other Unlimited Ammo - Enabled for {0}
-commands.unlimited.off.other Unlimited Ammo - Disabled for {0}
-commands.spraybypass.on Unlimited Ammo - Enabled
-commands.spraybypass.off Unlimited Ammo - Disabled
-commands.spraybypass.on.other Unlimited Ammo - Enabled for {0}
-commands.spraybypass.off.other Unlimited Ammo - Disabled for {0}
+commands.unlimited.on अनलिमिटेड Ammo - इनेबल्ड
+commands.unlimited.off अनलिमिटेड Ammo - डिसेबल्ड
+commands.unlimited.on.other अनलिमिटेड Ammo - {0} के लिए इनेबल्ड
+commands.unlimited.off.other अनलिमिटेड Ammo - {0} के लिए डिसेबल्ड
+commands.spraybypass.on अनलिमिटेड Ammo - इनेबल्ड
+commands.spraybypass.off अनलिमिटेड Ammo - डिसेबल्ड
+commands.spraybypass.on.other अनलिमिटेड Ammo - {0} के लिए इनेबल्ड
+commands.spraybypass.off.other अनलिमिटेड Ammo - {0} के लिए डिसेबल्ड
 commands.suspect.invalid Invalid suspect info
-commands.suspect.menu Suspect in menu
-commands.suspect.left Suspect has left
+commands.suspect.menu ससपेक्ट मेन्यू में है 
+commands.suspect.left ससपेक्ट ने मैच छोड़ दिया 
 commands.suspect.found Suspect Found. Spectating: {0}, Spawned: {1}, Name: {2}
-commands.forfeit.next Must play atleast 1 round
-commands.forfeit.wait Wait till match starts to forfeit
-commands.forfeit.later Please try again in a few seconds
-commands.forfeit.voted You already voted.
-commands.forfeit.limit You can only start one vote per game
-commands.forfeit.ended Forfeiting Game
+commands.forfeit.next 1 राउंड खेलना ज़रूरी है 
+commands.forfeit.wait forfeit करने के लिए मैच शुरू होने दें 
+commands.forfeit.later कृपया कुछ देर बाद कोशिश करें 
+commands.forfeit.voted आप पहले वोटे कर चुके हैं
```

Comment:
what is वोटे? should be **आप पहले ही वोट कर चुके हैं**

Reply:
another typo, typing with the microsoft indic tool messes up sometimes